<?xml version="1.0"?>
<launch>
    <arg name="est" default="ekf2"/>
    <!-- <arg name="world" default="$(find mavlink_sitl_gazebo)/worlds/outdoor1_light.world"/> -->
    <!-- <arg name="world" default="$(find mavlink_sitl_gazebo)/worlds/indoor1.world"/> -->
    <arg name="world" default="$(find px4_sim)/worlds/my_world.sdf"/>

    <!-- gz设置 gazebo configs -->
    <arg name="gui" default="true"/>
    <arg name="debug" default="false"/>
    <arg name="verbose" default="false"/>
    <arg name="paused" default="false"/>

    <!-- 打开gz加载世界 Gazebo sim -->
    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="gui" value="$(arg gui)"/>
        <arg name="world_name" value="$(arg world)"/>
        <arg name="debug" value="$(arg debug)"/>
        <arg name="verbose" value="$(arg verbose)"/>
        <arg name="paused" value="$(arg paused)"/>
    </include>

    <!-- 生成无人机 iris_0 -->

</launch>
<node pkg="tf" type="static_transform_publisher" name="iris_0_camera_link_to_depth"
        args="0.0 0.0 0 0.0 0.0 0.0 /camera_link /depth_camera_base 40" />
    <arg name="clip_distance"             default="-2"/>
    <arg name="use_rviz"                  default="false"/>
    <arg name="use_rtabmapviz"            default="false"/>
    <include file="$(find rtabmap_ros)/launch/rtabmap.launch">
        <arg name="rtabmap_args"       value="--delete_db_on_start"/>
        <arg name="depth_topic"        value="/iris_0/realsense/depth_camera/depth/image_raw"/>
        <arg name="frame_id"           value="camera_link"/>
        <arg name="visual_odometry"    value="false"/>
        <arg name="odom_topic"         value="/vins_estimator/odometry"/>
        <arg name="rgb_topic"          value="/iris_0/realsense/depth_camera/color/image_raw"/>
        <arg name="camera_info_topic"  value="/iris_0/realsense/depth_camera/color/camera_info"/>
        <arg name="queue_size"         value="200"/>
        <arg name="rviz"               value="$(arg use_rviz)"/>
        <arg name="rtabmapviz"         value="$(arg use_rtabmapviz)"/>
    </include>